mteb/tasks/Retrieval/eng/Touche2020Retrieval.py

```diff
@@ -57,3 +56,44 @@ class Touche2020(AbsTaskRetrieval):
             },
         },
     )
+
+
+class Touche2020v3(AbsTaskRetrieval):
+    metadata = TaskMetadata(
+        name="Touche2020v3",
```

Comment:
```suggestion
        name="Touche2020v3Retrieval",
```
To differentiate from the alternative: "Touche2020v3Retrieval.v3" (which signifies a version change with MTEB).